### PR TITLE
say verb support play audio in array

### PR DESCRIPTION
**Pull request by xquanluu**
https://github.com/jambonz/jambonz-feature-server/issues/472

### CODE REVIEW

File: lib/tasks/say.js [hunk 1]
```diff
@@ -153,7 +162,13 @@ class TaskSay extends Task {
         }
       };
 
-      const arr = this.text.map((t) => generateAudio(t));
+      const arr = this.text.map((t) => {
```
Comment (davehorton) [inline lib/tasks/say.js:165]: could we just make this
```
const arr = this.text.map((t) => this._validateURL(t) ? t : generateAudio(t);
```

**Review by davehorton — CHANGES_REQUESTED** [review]

**Comment by davehorton** [comment]
could you add a test case showing an array of mixed text and http urls to say

File: lib/tasks/say.js (hunk 1 above)
Comment (davehorton) [reply lib/tasks/say.js:165]: @xquanluu ?

Comment (xquanluu) [reply lib/tasks/say.js]: fixed

**Comment by xquanluu** [comment]
added testcase.

**Review by davehorton — APPROVED** [review]

**Pull request merged**